identity-admin/webapp/components/search/Pagination.test.tsx

```diff
@@ -0,0 +1,113 @@
+import Pagination from './Pagination';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { buildSearchUrl } from '../../utils/search-util';
+
+const renderComponent = (currentPage: number, pageCount: number) =>
+  render(<Pagination currentPage={currentPage} pageCount={pageCount} />);
+const getFirstLink = (container: HTMLElement) =>
+  container.querySelector(
+    '.user-pagination__item.user-pagination__item--first'
+  );
+const getLastLink = (container: HTMLElement) =>
+  container.querySelector('.user-pagination__item.user-pagination__item--last');
+const getAllLinks = (container: HTMLElement) =>
+  container.getElementsByClassName('user-pagination__item');
+const expectedLinkUrl = (expectedPage: string) =>
+  buildSearchUrl(expectedPage, 'active', 'Bob', 'bob@email.com', 'email', '1');
+
+jest.mock('next/router', () => ({
+  useRouter: () => {
+    return {
+      query: {
+        status: 'active',
+        name: 'Bob',
+        email: 'bob@email.com',
+        sort: 'email',
+        sortDir: '1',
+      },
+    };
+  },
+}));
+
+describe('Pagination', () => {
+  it('disables previous link on first page', async () => {
```

Comment:
Tests don't need to be `async` by default. Only certain react-testing-library functions are `async`, primarily `waitFor` and `findBy*`.